### PR TITLE
CI for airflow source

### PR DESCRIPTION
**Pull request by laysabit**
<details>
  <summary>PR Checklist</summary>

### PR Structure

- [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
- [x] This PR avoids mixing refactoring changes with feature changes (split into two PRs
      otherwise).

### Thoroughness

- [x] This PR adds tests for the most critical parts of the new functionality or fixes.
- [ ] I've updated the docs and README with the added features, breaking changes, new instructions on how to use the repository.

### Release planning

- [x] I've decided if this PR requires a new major/minor/patch version accordingly to
    [semver](https://semver.org/), and I've changed the name of the BRANCH to release/* , feature/* or patch/* .
</details>

### What

This PR add a new source, Airflow, to the integration tests.

### Why

With that new source we can build the models related to airflow on CI to analyse if the changes being made are functional, successful.

### Known limitations

[N/A]

### CODE REVIEW

File: change_dbt_project_airflow_source.sh
Comment (guizsantos) [inline change_dbt_project_airflow_source.sh:1]: I would recommend the usage of `yq` for data manipulation of YAML! files, it is designed for exactly such purposes.

File: integration_tests/seeds/airflow/dag_run.csv [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-5678,dag_tags_rbac,2023-08-29T20:30:00Z,success,scheduled__2023-08-29T20:30:00+00:00,FALSE,80047d942e,2023-08-29T21:00:49.962769Z,2023-08-29T21:00:00.544975Z,scheduled,2023-08-29T21:00:49.952562Z,3616896069a7d5a3b40f4478372f03da,10415,2023-08-29T21:00:00.452329Z,2023-08-29T20:30:00Z,2023-08-29T21:00:00Z,1,2023-08-29T21:00:49.96384Z,2024-02-02T11:04:16.362959Z,2024-02-02T11:04:30.575128Z,2024-02-02T11:04:17.37818Z,,1706871870575,
\ No newline at end of file
+"id","dag_id","execution_date","state","run_id","external_trigger","conf","end_date","start_date","run_type","last_scheduling_decision","dag_hash","creating_job_id","queued_at","data_interval_start","data_interval_end","log_template_id","updated_at"
+87755,airflow_ng,2024-07-04 10:56:00.000 -0300,success,scheduled__2024-07-04T13:56:00+00:00,false,�\u0004}�.,2024-07-04 10:57:15.119 -0300,2024-07-04 10:57:00.256 -0300,scheduled,2024-07-04 10:57:15.109 -0300,"2606cfccb8540961ee80c09fe32dcc8d",110841,2024-07-04 10:57:00.210 -0300,2024-07-04 10:56:00.000 -0300,2024-07-04 10:57:00.000 -0300,1,2024-07-04 10:57:15.120 -0300
```
Comment (guizsantos) [inline integration_tests/seeds/airflow/dag_run.csv:2]: It looks like this CSV file was not properly encoded.

File: integration_tests/seeds/airflow/task_instance.csv [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-MOODLE.mdl_grade_items,moodle_training_dag,scheduled__2023-12-27T00:00:00+00:00,2023-12-28T00:16:24.451079Z,2023-12-28T00:17:47.59865Z,83,success,1,moodle-training-dag-moodle-mdl-43ba21e929bc488c83ce96a5c83b016b,root,38309,meltano_pool,default,1,DockerOperator,2023-12-28T00:15:13.134265Z,22,2,80047d942e,1,38299,,,,,null,-1,2023-12-28T00:16:24.661501Z,2024-02-02T11:02:10.659834Z,2024-02-02T11:02:36.213409Z,2024-02-02T11:02:21.062529Z,,1706871756213,
\ No newline at end of file
+"task_id","dag_id","run_id","start_date","end_date","duration","state","try_number","hostname","unixname","job_id","pool","queue","priority_weight","operator","queued_dttm","pid","max_tries","executor_config","pool_slots","queued_by_job_id","external_executor_id","trigger_id","trigger_timeout","next_method","next_kwargs","map_index","updated_at"
+dbt_source_test,dbt_bitrix,scheduled__2023-01-12T06:00:00+00:00,2023-01-13 03:01:07.644 -0300,2023-01-13 03:01:27.852 -0300,20.207217,success,1,dbtbitrixdbtsourcetest-32bc3af501374e48913fad10b54fdd67,root,332,default_pool,default,17,DockerOperator,2023-01-13 03:00:58.037 -0300,21,2,�\u0004}�.,1,201,,,,,"",-1,
```
Comment (guizsantos) [inline integration_tests/seeds/airflow/task_instance.csv:2]: Same thing, it looks like this CSV file was not properly encoded.

**Review by guizsantos — COMMENTED** [review]
Looks ok to me. Comments are for improving only.

**Review by vitoravancini — APPROVED** [review]

**Pull request merged**